calour/heatmap/heatmap.py

```diff
@@ -169,46 +169,47 @@ def heatmap(exp: Experiment, sample_field=None, feature_field=None,
             xticklabel_kwargs=None, yticklabel_kwargs=None,
             xticklabel_len=16, yticklabel_len=16,
             xticks_max=10, yticks_max=30,
-            clim=(None, None), cmap='viridis', norm=mpl.colors.LogNorm(),
-            title=None, rect=None, cax=None, ax=None):
+            norm=None, clim=(None, None), cmap='viridis',
```

Comment:
why change the default norm from log scale? log is useful for most cases.

Reply:
for both ampliconexperiment and ms1experiment we need norm=log by default. where is it set?